Content/.gitignore

```diff
@@ -1,16 +1,362 @@
-.fable/
-.fake/
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
As this is the "minimal" option, I'd keep everything as minimal as possible - including gitignore
As an "expert" wanting to have everything in control I'd not be happy with a full-blown gitignore file

Reply:
That's a good point. Perhaps just create two different gitignore files?

Reply:
Not sure if there's a need to have the auto-generated gitignore, even for full template - any specific reasons there?

Reply:
I assumed that a standard one will catch most elements that may crop up in the future once people start working on a project and not need to add to the file. Happy to revert.

Reply:
For VS I think you only need `.vs/` these days. I am against having these uber gitignores.

I might even go so far as to say it's up to the user to manage this in their user-level gitignore because they could be using any IDE, and we should only ignore things inherent to the project like `node_modules` etc. But I'm OK with covering VS, VS Code, Ionide and Rider which should only be 4 lines.

Reply:
There are surely some others as well - bin and obj etc. etc.?

Reply:
I was only referring to the ones that come from your choice of IDE/editor.

Was there a problem with the existing gitignore? It had 16 lines and covered everything we needed. It would work perfectly for the full/opinionated template option too.